### DISCUSSION TITLE
If you lower the fan speed in custom mode, it will not work.

### DISCUSSION THREAD

**Discussion by eotaekyong** [BartoszCichecki/LenovoLegionToolkit#1071] [Q&A] [answered]
Model: Legion Slim 5 14APH8
BIOS version: MACN23WW
Lenovo Region Toolkit Version: 2.18.0

hello
The fan speed is too noisy and I would like to lower the fan speed.
I set the fan speed to a maximum of 3300 in power custom mode, but in actual gaming, it goes up to over 4000. It doesn't seem to work.

Is there any way to solve this?

![스크린샷 2023-12-10 002002](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/13896133/e55e628f-9e3c-42c3-b442-c1d9c7c297f5)
![1](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/13896133/e010b6c7-88b2-4694-8284-7e84bb40df0d)

**Comment by BartoszCichecki** [comment, marked as answer]
This is up to Lenovo whether they allow such settings or not - in this case apparently EC doesn't respect lower fan speeds.
